index.html

```diff
@@ -54,10 +54,10 @@ <h1>I'm Zermarden <br>
         <img class="header-background" src="assets/header-background-mobile.svg" alt="background-image">
```

Comment:
- [ ] I kindly suggest that you use the background-image CSS property to add your background images because they are only used for decorations